lib/quiq/client.rb

```diff
@@ -5,15 +5,20 @@
 
 module Quiq
   class Client
-    def push(job)
+    def push(job, scheduled_at)
       serialized = JSON.dump(job.serialize)
+
       Async do
-        Quiq.redis.lpush("queue:#{job.queue_name}", serialized)
+        if scheduled_at
```

Comment:
I think it should be better to test if scheduled_at > now() here. Wdyt ?

Reply:
In this case it will never process some jobs that were scheduled while server was down.